yaptide/application.py

```diff
@@ -12,12 +14,20 @@ def create_app():
     """Function starting Flask Server"""
     flask_name = __name__.split('.')[0]
     app = Flask(flask_name)
+    
+    logstash_host = "logstash"
+    logstash_port = 5001
+    logstash_handler = logstash.TCPLogstashHandler(logstash_host, logstash_port, version=1)
+    logstash_handler.setLevel(logging.DEBUG)
```

Comment:
logging level should be parametrized somehow and controlled for example by env variables. It's not the best idea to have DEBUG level on production.